modules/ocf/templates/firefox/prefs.js.erb

```diff
@@ -0,0 +1,12 @@
+lockPref("browser.startup.homepage", "<%= @desktop_homepage %>");
+pref("browser.cache.disk.capacity", 0);
+pref("browser.download.useDownloadDir", false);
```

Comment:
(i realize this is just a renamed file in this PR, though. don't let this block merging.)